python/paddle/fluid/layers/nn.py

```diff
@@ -10561,3 +10562,50 @@ def tree_conv(nodes_vector,
     else:
         pre_activation = out
     return helper.append_activation(pre_activation)
+
+
+def cvm(input, cvm, use_cvm=True):
+    """
+    **cvm layers**
+
+    This layer accepts a tensor named input which is ID after embedded and lod level is 1 ,
+         cvm is a show_click info and returns the cvm output.
```

Comment:
not sure what does "cvm is a show_click info and returns the cvm output" mean. what is `cvm`?

Reply:
cvm is continuous value moded. now, it only consider show and click value in ctr project.